src/esf-mail/esf-mail-notification.ts

```diff
@@ -0,0 +1,62 @@
+import * as core from 'aws-cdk-lib';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { RetentionDays } from 'aws-cdk-lib/aws-logs';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import * as ssm from 'aws-cdk-lib/aws-ssm';
+import { Construct } from 'constructs';
+import { statics } from '../statics';
+
+/**
+ * Sends email notifications to mailbox (Nijmegenaar) when certain ESF is ready to be filled in
+ */
+export function setupEsfNotificationMail(
+  scope: Construct,
+  domainName: string,
+) {
+
+  const esfMailNotificationLambda = new lambda.Function(scope, 'esf-mail-notification-lambda', {
+    description: 'Lambda to send email notifications to mailbox when a specific ESF is available',
+    runtime: lambda.Runtime.PYTHON_3_9,
+    handler: 'index.lambda_handler',
+    code: lambda.Code.fromAsset('src/esf-mail/esf-mail-notification-lambda'),
+    logRetention: RetentionDays.ONE_MONTH,
```

Comment:
lijkt me verstandiger om de logs langer te bewaren, 6maand bijvoorbeeld, omdat er altijd met vertraging vragen komen of mails wel verstuurd zijn.
Daarop aansluitend: het lijkt me verstandig om de logging zo te maken dat die goed gestructureerd is en goed met querys doorzocht kan worden, zodat heel snel te achterhalen is of en wanneer voor een specifiek mailadres mails verstuurd zijn, of wat er bij een specifiek batch misgegaan is; het lijkt me ook verstandig om te zorgen dat deze info op een of andere manier bij de applicatiebeheerder terecht kan komen, om te voorkomen dat wij verzoeken krijgen om dit soort info boven tafel te krijgen. Ik durf er wel wat op te zetten dat dit soort vragen gaan  komen namelijk.

Reply:
Prima, neem ik die wijzigingen mee in de volgende PR.